book/getting_started/running.md

```diff
@@ -0,0 +1,147 @@
+# Running Reth
+
+This chapter will provide a few runbooks for getting the node up and running.
+
+This goal of this is not to expose all of the options available to you for running Reth (e.g. config file options, CLI flags, etc.) - those will be documented in the other chapters - but rather showcase some examples of different ways in which one could run the node.
+
+## Run on MacOS w/ Prometheus + Grafana for Metrics
+
+This will provide a brief overview for how to run Reth locally on a Mac.
+
+First, ensure that you have Reth installed by following the [instructions to install on Mac][mac-installation].
+
+### Basic operation
+
+The most basic way to run it is by using the following command:
+
+```bash
+cargo run --release -- node
+```
+
+This will build Reth using cargo's [release profile][release-profile], and run it with an error log level by default.
+
+You will likely see a large number of error logs to the tune of:
+
+```bash
+ERROR Disconnected by peer during handshake: Too many peers
+```
+
+These simply indicate that the peer you discovered is already connected to too many other peers and cannot accept a new connection with you. Nothing to worry about here, this is part of the peering process.
+
+Believe it or not, your node is already running and attempting to sync.
+
+When playing around with sync, you may want to set a threshold block that you'd like to sync up to instead of going for a full sync from the jump.
+
+To do so, add the `--debug.tip` flag with the block hash that you'd like to sync to:
+
+```bash
+cargo run --release -- node --debug.tip 0x8e38b4dbf6b11fcc3b9dee84fb7986e29ca0a02cecd8977c161ff7333329681e
+```
+
+This will sync to block 1M.
+
+That's great and all, but wouldn't it be nice to get a deeper view into what's going on?
+
+### Adding more logs to the mix
+
+Let's try again, now with the following command:
+
+```bash
+RUST_LOG=info,sync::stages=trace,downloaders=trace cargo run --release -- node --debug.tip 0x8e38b4dbf6b11fcc3b9dee84fb7986e29ca0a02cecd8977c161ff7333329681e
+```
+
+This will dump info-level logs throughout the node's operation, as well as trace-level logs for the pipeline stages and the downloaders (which fetch data over the P2P network). Check out the [docs][docs] for more info on these!
+
+Now, you'll probably see a _lot_ of warning logs stemming from attempting to connect to peers, including but not limited to:
+    - `error=Some(Eth(P2PStreamError(HandshakeError(NoResponse))))`
+        - This means you hit a timeout when trying to do a devp2p handshake with the given peer
```

Comment:
I believe that's not right, and it's just a peer simply not responding with data? @onbjerg

Reply:
ah, i assumed not responding with data would mean we waited too long and got no response

Reply:
There's a dedicated variant for timeouts, no response is that they just sent us no data